ArrayFreqHash.rb

```diff
@@ -0,0 +1,19 @@
+print "Enter Size Of Array : "
+n = gets().chomp().to_i
```

Comment:
1. Use meaning full variable names
2. No need to pass `()` if there are no method paramters 
```suggestion
n = gets.chomp.to_i
```